src/styles/main.scss

```
@@ -177,11 +183,56 @@ h1 {
 }
 
 .message-win {
-  background: #edc22e;
+  background: #2669b5;
   color: #f9f6f2;
 }
 
+.message-lose {
+  width: 100%;
+  box-sizing: border-box;
+  height: 60px !important;
```

Don't use important, it's a bad practice
```suggestion
  height: 60px;
```